src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -909,11 +914,16 @@
         - name: --enable-encryption-at-host
           type: bool
           short-summary: Enable EncryptionAtHost on agent node pool.
+        - name: --ossku
+          type: string
+          short-summary: The ossku of the agent node pool.
     examples:
         - name: Create a nodepool in an existing AKS cluster with ephemeral os enabled.
           text: az aks nodepool add -g MyResourceGroup -n nodepool1 --cluster-name MyManagedCluster --node-osdisk-type Ephemeral --node-osdisk-size 48
         - name: Create a nodepool with EncryptionAtHost enabled.
           text: az aks nodepool add -g MyResourceGroup -n nodepool1 --cluster-name MyManagedCluster --enable-encryption-at-host
+        - name: Create a nodepool cluster with a specific ossku
+          text: az aks nodepool add -g MyResourceGroup -n nodepool1 --cluster-name MyManagedCluster  --ossku CBLMariner
```

Comment:
let's add a comment here that CLBMariner is for MS tenant usage only. Or doesn't use CBLMariner example at all

Reply:
Switched the examples to Ubuntu which will work on every subscription.